CITATION.cff

```diff
@@ -26,33 +26,51 @@ authors:
     orcid: https://orcid.org/0000-0002-1885-4405
   - family-names: Pei
     given-names: Ruifan
+    orcid: https://orcid.org/0009-0003-4964-2821
   - family-names: Arevalo
     given-names: John
     orcid: https://orcid.org/0000-0002-1138-5036
   - family-names: Tsang
     given-names: Hillary
+    orcid: https://orcid.org/0000-0001-6461-3402
   - family-names: Rubinetti
     given-names: Vincent
+    orcid: https://orcid.org/0000-0002-4655-3773
+  - family-names: Tromans-Coia
+    given-names: Callum
+    orcid: https://orcid.org/0000-0002-5518-8915
+  - family-names: Becker
+    given-names: Tim
+    orcid: https://orcid.org/0000-0001-9615-0799
   - family-names: Weisbart
     given-names: Erin
     orcid: https://orcid.org/0000-0002-6437-2458
   - family-names: Bunne
     given-names: Charlotte
+    orcid: https://orcid.org/0000-0003-1431-103X
   - family-names: Kalinin
     given-names: Alexandr A.
+    orcid: https://orcid.org/0000-0003-4563-3226
   - family-names: Senft
     given-names: Rebecca
+    orcid: https://orcid.org/0000-0003-0081-4170
   - family-names: Taylor
     given-names: Stephen J.
     orcid: https://orcid.org/0000-0002-7279-7798
   - family-names: Jamali
     given-names: Nasim
+    orcid: https://orcid.org/0000-0003-1851-6585
   - family-names: Adeboye
     given-names: Adeniyi
+  - family-names: Abbasi
```

Comment:
Can you make sure the ordering aligns with the paper?

Thanks for also adding the ORCID! Can you double check all are correct? It's very important! (please use the existing author spreadsheet)

Reply:
(e.g., Hamdah is out of order here)

Reply:
Reviewed both ORCID and author order list with the csv author list and the authors posted on the pre-print. The order is correct!